week-4/C-more-complex-objects/exercise-2.js

```diff
@@ -28,15 +28,22 @@ var newCurrentOwner = {
 // - assign the value of the variable 'newCurrentOwner' as the value to the house's "currentOwner"
 // - from the list of previous owners, replace only "John A." with "Stephen B."
 // - give the house a new property called 'isForSale' with the value 'false'
-
-
-
-
+newCurrentOwner.firstName = "Margaret";

```

Comment:
We want Georgina Hernandez to be the `house.currentOwner`.  Also, no need to assign it string by string.